### PR TITLE
ef -  toggle wheelchair backend

### PR DESCRIPTION
**Pull request by EKF626**
In this PR we give backend functionality to tell whether a user needs a wheelchair and to toggle that boolean value. Added this parameter to the user class, added the CRUD operation, and added test cases. To test, use swagger to toggle the wheelchair parameter of a user.

### CODE REVIEW

**Review by cmfisher606 — DISMISSED** [review]
Lgtm. Tests seem valid

File: src/main/java/edu/ucsb/cs156/gauchoride/controllers/UsersController.java [hunk 1]
```diff
@@ -156,4 +156,20 @@ public Object toggleRider( @ApiParam("id") @RequestParam Long id){
         }
     }
 
+    @ApiOperation(value = "Toggle the wheelchair field")
+    @PostMapping("/toggleWheelchair")
+    public Object toggleWheelchair( @ApiParam("id") @RequestParam Long id){
```
Comment (andrewpengucsb) [inline src/main/java/edu/ucsb/cs156/gauchoride/controllers/UsersController.java:161]: Code logic seems fine. However if the intention is to have this admin only then this is the correct file to put it in but it needs the preauth for role admin. If the intention is to give users the ability to toggle their own wheelchair status then it appears to be a mismatch to be located in the admin only controller.

**Review by andrewpengucsb — DISMISSED** [review]

**Review by pconrad — APPROVED** [review]
LGTM

**Review by pconrad — APPROVED** [review]
LGTM

**Pull request merged**